opencl/cryptonight.cl

```diff
@@ -13,7 +13,36 @@
   * along with this program.  If not, see <http://www.gnu.org/licenses/>.
   */
 
+#ifdef cl_amd_media_ops
+#pragma OPENCL EXTENSION cl_amd_media_ops : enable
+#else
+#pragma "Emulating amd_bitalign"
+// taken from https://github.com/Bdot42/mfakto/blob/master/src/common.cl
+// we should define something for bitalign() ...
+//     Build-in Function
+//      uintn  amd_bitalign (uintn src0, uintn src1, uintn src2)
+//    Description
+//      dst.s0 =  (uint) (((((long)src0.s0) << 32) | (long)src1.s0) >> (src2.s0 & 31))
+//      similar operation applied to other components of the vectors.
+#define amd_bitalign(src0, src1, src2) (src0  << (32-src2)) | (src1 >> src2)
```

Comment:
BUG: the parentheses are wrong placed and missing cast to 64 bit: this is solve din #17